eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java

```diff
@@ -58,11 +58,11 @@
                     try (BufferedInputStream bis = new BufferedInputStream(System.in);
                         ByteArrayOutputStream buf = new ByteArrayOutputStream()) {
                         while (true) {
-                            final int b = bis.read();
-                            if (b == -1) {
+                            final int byteData = bis.read();
```

Comment:
@alex-semenyuk Can you make this name not compound? Please, read this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
Yeah, I read this, but I think here more suitable is byte, but it's java keyword.

Reply:
@alex-semenyuk I think `data` is good enough